src/lib/routeLoaders.ts

```diff
@@ -184,21 +180,7 @@ export const homeLoader: LoaderFunction = async (): Promise<
   if (!isDesktop()) {
     return redirect(PATHS.FILE + '/%2F' + BROWSER_PROJECT_NAME)
   }
-  const { configuration } = await loadAndValidateSettings()
-
-  const projectDir = await ensureProjectDirectoryExists(configuration)
-
-  if (projectDir) {
-    const projects = await listProjects(configuration)
-
-    return {
-      projects,
-    }
-  } else {
-    return {
-      projects: [],
-    }
-  }
+  return {}
```

Comment:
Oh interesting why not just strip out homeLoader, trying to keep the PR size down? Can we add a `TODO` comment?

Reply:
Well, we need the this router loader still for the check at the top.

Additionally we may find ourselves needing this router loader again, so no harm leaving it I figured